build.gradle

```diff
@@ -6,6 +6,16 @@ buildscript {
     google()
     mavenCentral()
     jcenter()
+    maven {
+      url 'https://api.mapbox.com/downloads/v2/releases/maven'
+      authentication {
+        basic(BasicAuthentication)
+      }
+      credentials {
+        username = "mapbox"
+        password = project.hasProperty('SDK_REGISTRY_TOKEN') ? project.property('SDK_REGISTRY_TOKEN') : System.getenv('SDK_REGISTRY_TOKEN')
```

Comment:
Could you add a note about this requirement to https://github.com/mapbox/mapbox-java/blob/master/INSTALL.md? You can copy from https://github.com/mapbox/mapbox-navigation-android/blob/main/CONTRIBUTING.md.

Reply:
👍  perfect, updated